src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -65,6 +66,11 @@ public class JwtTokenUtilities
             JwtHeaderParameterNames.Zip
         };
 
+        /// <summary>
+        /// Creates a dictionary from set of Claim's 
+        /// </summary>
+        /// <param name="claims"></param>
+        /// <returns></returns>
```

Comment:
Returns section should be filled out

Reply:
Good catch.

Reply:
<param> section should be filled out as well.